site/profile/data/common.yaml

```diff
@@ -5,6 +5,7 @@ profile::core::convenience::packages:
   - "neovim"
   - "tree"
   - "vim"
+  - "screen"
```

Comment:
Please sort the packages list. The reason to keep lists sorted is to try to avoid merge conflicts down the road. It is impossible to completely avoid the problem but it is slightly less likely to happen with sorted lists. In vim/nvim this is easy to do with <visual block> then `!sort`.